Mathlib/AlgebraicGeometry/Morphisms/Basic.lean

```diff
@@ -236,11 +232,9 @@ theorem targetAffineLocallyOfOpenCover {P : AffineTargetMorphismProperty} (hP :
     exact (hP.1.arrow_mk_iso_iff (morphismRestrictOpensRange f _)).mpr h𝒰
 #align algebraic_geometry.target_affine_locally_of_open_cover AlgebraicGeometry.targetAffineLocallyOfOpenCover
 
--- set_option maxHeartbeats 1000000 in
-open List in
 theorem AffineTargetMorphismProperty.IsLocal.affine_openCover_TFAE
     {P : AffineTargetMorphismProperty} (hP : P.IsLocal) {X Y : Scheme.{u}} (f : X ⟶ Y) :
-    TFAE
+    List.TFAE
```

Comment:
Why this change? I like it if the statement is just "The Following Are Equivalent"...
Do you need this for a technical reason?

Reply:
It saves on line count. `open List in` ends up being used for just this one `TFAE`.

Reply:
Hmm, I don't find that a very compelling reason.